frontend/app/views/joiner/thankyou.scala.html

```diff
@@ -62,21 +62,30 @@ <h2 class="page-section__headline">Your membership summary</h2>
                         <td id="qa-joiner-summary-today">@summary.amountPaidToday.pretty</td>
                     </tr>
                     <tr role="row">
-                        <th role="rowheader">
-                            @if(summary.annual) { Annual } else { Monthly } payment
-                        </th>
-                        <td id="qa-joiner-summary-recurring">@summary.planAmount.pretty</td>
-                    </tr>
-                    <tr role="row">
                         <th role="rowheader">Next payment</th>
                         <td id="qa-joiner-summary-next">
-                        @if(summary.initialFreePeriodOffer) {
+                            @if(summary.initialFreePeriodOffer) {
                             @summary.nextPaymentPrice.pretty (@summary.nextPaymentDate.pretty)
```

Comment:
Can I be a bit pedantic and ask you to indent these bits? I'm too lazy to actually read code and instead rely on indents to understand what's happening.